### PR TITLE
ZER-138 update README.md

### PR DESCRIPTION
**Pull request by KostiaFed**
dev
## JIRA

* [ZER-138](https://jira.softserve.academy/browse/ZER-138)

ToDo

Create meaningful and detailed documentation for local setup.

Documentation that will include:

How install and configure postgresql.
How install and configure yarn packages.
Explanation of config files for local setup.
Redis
Common system errors fixes (windows, linux etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,18 +37,104 @@ The latest version from the release branch 'master' is automatically deployed to
...
+<a href="https://redis.io/docs/getting-started/">Install Redis</a> for your operating system or subsystem. You can familiarize yourself with
+<a href="https://redis.io/docs//">Redis documentation</a>.
+
+Installation for Ubuntu:
+
+$ `curl -fsSL https://packages.redis.io/gpg | sudo gpg --dearmor -o /usr/share/keyrings/redis-archive-keyring.gpg`
+
+$ `echo "deb [signed-by=/usr/share/keyrings/redis-archive-keyring.gpg] https://packages.redis.io/deb $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/redis.list`
+
+$ `sudo apt-get update`
```
Comment (opazyniuk) [inline README.md:94]: please try to add ability to copy commands from option from the right from command

File: README.md [hunk 2]
```diff
@@ -37,18 +37,104 @@ The latest version from the release branch 'master' is automatically deployed to
...
+<a href="https://github.com/mperham/sidekiq">Sidekiq documentation</a>.
+
+Installation:
+$ `bundle add sidekiq`
+
 ## How to run local
 
 1. Open terminal.
-2. Run `rails server` to start application.
+In some systems, after restarting them, the postgresql server remains disabled, perhaps at the first start you should enter "sudo service postgresql start".
```
Comment (opazyniuk) [inline README.md:131]:
```suggestion
In some systems, after restarting them, the postgresql server remains disabled, perhaps at the first start you should enter `sudo service postgresql start`.
```

**Review by opazyniuk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KostiaFed) [reply README.md:94]: I`m not sure that is possible

**Review by opazyniuk — APPROVED** [review]

**Review by denysiuk — APPROVED** [review]
Everything looks good, well done

**Review by anastasiiahryniv — APPROVED** [review]

**Pull request merged**